src/VisualStudio/Core/Impl/CodeModel/InternalElements/CodeParameter.cs

```diff
@@ -62,6 +62,17 @@ protected override EnvDTE.CodeElements GetCollection()
             return GetCollection<CodeParameter>(Parent);
         }
 
+        protected override string GetName()
+        {
+            return this._name;
+        }
+
+        protected override string GetFullName()
+        {
+            var node = LookupNode();
```

Comment:
GetParameterFullName() will also throw an InvalidOperationException if node is null. However, before that happens, an E_FAIL will be thrown by LookupNode(). I think that's probably the right thing in this case.